src/core/wappu-mood-core.js

```diff
@@ -240,6 +241,77 @@ function _hasDescription(row) {
   return row.description && _.trim(row.description).length > 0;
 }
 
+function _interpolateIntermediate(moods) {
+  const getIndexOfToday = array => {
+    const today = moment().utcOffset(3); // Hacky
+
+    return _.findIndex(array, element =>
+      moment(element.date).isSame(today, 'day')
+    ) || 0;
+  };
+
+  const todayIndex = getIndexOfToday(moods);
+  const datesPast = _fillEmpties(moods.slice(0, todayIndex + 1));
+  const datesFuture = moods.slice(todayIndex + 1);
+
+  return  _.union(datesPast, datesFuture);
+}
+
+function _fillEmpties(dates) {
+
+  if (dates.length <= 1) {
+    // Too small sample size to interpolate missing values
+    return dates;
+  }
+
+  const interpolateCity = !!dates[0].ratingCity;
+  const interpolateTeam = !!dates[0].ratingTeam;
+  const interpolatePersonal = !!dates[0].ratingPersonal;
```

Comment:
IMHO `!!` is a bit hacky way to coerce a value to boolean, `Boolean(dates[0].ratingCity)` is a bit more readable